HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,182 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
```

Comment:
Remove all help text that I gave you, so the file looks cleanly like your contribution.